src/main/java/com/epam/javacourses2016/task7/SolverTask7.java

```diff
@@ -12,11 +13,25 @@
 
     /**
      * Осуществляет перемножение двух многочленов.
-     * @param first Первый многочлен.
+     *
+     * @param first  Первый многочлен.
      * @param second Второй многочлен.
      * @return Многочлен, полученный в результате перемножения.
      */
     public List<Integer> multiplyPolynomials(List<Integer> first, List<Integer> second) {
-        return null;
+        List<Integer> result = new ArrayList<>();
```

Comment:
Better use Map there, HashMap is fast, O(1) on access - like ArrayList but has more readability